packages/ra-input-rich-text/package.json

```diff
@@ -46,6 +46,7 @@
         "react-dom": "^16.9.0"
     },
     "dependencies": {
+        "@material-ui/core": "^4.11.0",
```

Comment:
not good: the ra-ui-material-ui dependency is ^4.3.3. This is a breaking change. 

Why did you need an earlier version in the first place?

Reply:
I actually didn't change that. Linters were throwing error of `@material-ui/core` not being installed, So I installed it.